core/domain/stats_services_test.py

```diff
@@ -327,29 +372,42 @@ class UnresolvedAnswersTests(test_utils.GenericTestBase):
     """Test the unresolved answers methods."""
 
     DEFAULT_RULESPEC_STR = exp_domain.DEFAULT_RULESPEC_STR
+    DEFAULT_SESSION_ID = 'session_id'
+    DEFAULT_TIME_SPENT = 5.0
+    DEFAULT_PARAMS = {}
 
     def test_get_top_unresolved_answers(self):
+
```

Comment:
Nit: please remove blank line between function declaration and the first line of code (to be consistent with other test functions in this file).